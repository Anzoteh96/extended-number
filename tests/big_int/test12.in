-2587460687961586246986992829738874696
-960295182919273681699768649936150246
