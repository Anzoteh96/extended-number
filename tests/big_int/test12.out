Enter two numbers n and m:
n is -2587460687961586246986992829738874696
m is -960295182919273681699768649936150246
n + m is -3547755870880859928686761479675024942
n - m is -1627165505042312565287224179802724450
n * m is 2484726034642501186934934879311237477886812779295825441160489366423575216
n / m is 2
n % m is -666870322123038883587455529866574204
